doozer/tests/backend/test_konflux_olm_bundler.py

```
@@ -131,6 +131,82 @@ async def test_replace_image_references(self, mock_oc_image_info):
+        mock_oc_image_info.return_value = mock_image_info
+
+        # Mock group_config with MAJOR=4
+        self.rebaser._group_config.get.side_effect = lambda key, default=None: {
+            'csv_namespace': 'namespace',
+            'operator_image_ref_mode': 'manifest-list'
+        }.get(key, default)
+        self.rebaser._group_config.vars = {'MAJOR': 4}
+
+        metadata = MagicMock()
+        metadata.runtime.group = "openshift-4.19"
+        metadata.runtime.data_dir = "/tmp/nonexistent"
```

Ruff (0.15.20)

[error] Probable insecure usage of temporary file or directory: "/tmp/nonexistent"

(S108)